resources/styles.css

```diff
@@ -53,6 +53,39 @@
     vertical-align: middle;
 }
 
+.tooltip {
+  position: relative;
+  height: 42px;
+  width: 30px;
+}
+
+.tooltip .tooltip-text {
+  visibility: hidden;
+  position: absolute;
+  background-color: black;
+  color: white;
+  text-align: center;
+  z-index: 997;
+  bottom: 40px;
+  right: 0;
+  border-radius: 3px;
+  padding: 0 10px;
+}
+
+.tooltip .tooltip-text::after {
+  content: '';
+  position: absolute;
+  top: 100%;
+  right: 10px;
+  border-width: 5px;
+  border-style: solid;
+  border-color: black transparent transparent transparent;
+}
+
+.tooltip:hover .tooltip-text {
```

Comment:
Have you tried using a tooltip defined in materialize to improve the behavior of opening and closing? (some kind of animation or something?)